doc/ref_model/chapters/chapter05.rst

```diff
@@ -1,24 +1,25 @@
-Feature set and Requirements from Infrastructure
+Feature Set and Requirements from Infrastructure
 ================================================
 
 A profile :ref:`chapters/chapter02:profiles, profile extensions & flavours` specifies the configuration of a
-Cloud Infrastructure node (host or server); :ref:`chapters/chapter02:profile extensions (specialisations)`
-may specify additional configuration. Workloads utilise profiles to describe the configuration of nodes on which they
-can be hosted to execute on. Workload Flavours provide a mechanism to specify the VM or Pod sizing information to host
-the workload. Depending on the requirements of the workloads, a VM or a Pod will be deployed as per the specified
-Flavour information on a node configured as per the specified Profile. Not only do the nodes (the hardware) have to be
-configured but some of the capabilities also need to be configured in the software layers (such as Operating System and
-Virtualisation Software). Thus, a Profile can be defined in terms of configuration needed in the software layers, the
-Cloud Infrastructure Software Profile, and the hardware, the Cloud Infrastructure Hardware Profile.
-
-Cloud Infrastructure Software profile description
+Cloud Infrastructure node (host or server). :ref:`chapters/chapter02:profile extensions (specialisations)`
+may specify additional configurations. Workloads utilise profiles to describe the configuration of nodes on which they
```

Comment:
replace utilise with use

Reply:
fixed in https://github.com/anuket-project/RM/pull/17